eb_hooks.py

```diff
@@ -942,6 +942,19 @@ def pre_configure_hook_LAMMPS_zen4(self, *args, **kwargs):
     else:
         raise EasyBuildError("LAMMPS-specific hook triggered for non-LAMMPS easyconfig?!")
 
+def pre_configure_hook_cmake_system(self, *args, **kwargs):
+    """
+    pre-configure hook for cmake built with SYSTEM toolchain:
+    - unset configopts for cmake easyconfigs using the SYSTEM toolchain
+    - https://github.com/EESSI/software-layer/issues/1175
+    """
+
+    if self.name == 'CMake':
+        if self.toolchain.name == 'system':
+            print_msg("Unset configopts to use ncurses library from the EESSI compatibility layer")
```

Comment:
Use `self.log.info` here, or `print_msg` with `log=self.log` if you want to be very verbose about this.

I would also log what the previous value of `configopts` was, so it's clear what's being wiped.

Reply:
@boegel I have updated the log messages as suggested